Dockerfile

```diff
@@ -34,10 +36,32 @@ RUN apt-get update -qq && \
     memcached \
     postfix \
     postgresql \
+    $PGLOADER_DEPENDENCIES \
     apache2 \
     supervisor && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
+###->-### install pgloader
+
+# install Clozure CL to avoid memory issues with the standard SBCL
```

Comment:
Might be useful to put that behind a build-arg maybe? To avoid unnecessary stuff in the SaaS image?

Reply:
We have a separate file for the SaaS docker image so that is not a problem. But I can still add the build arg.

Reply:
Right, that's fine then